textbook/styles.css

```
@@ -288,3 +291,44 @@ h4 {
   height: auto !important;
 }
 
+/* Callout styling */
+
+.callout-ex-prompt {
+  border-left-color: #2780e3 !important;
+}
+.callout-ex-prompt > .callout-header {
+  background-color: #ecf4fc !important;
+}
+.callout-ex-prompt > .callout-header::before {
+  font-family: "Font Awesome 5 Free";
+  content: "\f303";
+  margin-right: 5px;
+  font-size: 13px;
+}
+
+.callout-ex-hint {
+  border-left-color: #f0ad4e !important;
+}
+.callout-ex-hint > .callout-header {
+  background-color: #fef8ef !important;
+}
+.callout-ex-hint > .callout-header::before {
+  font-family: "Font Awesome 5 Free";
+  content: "\f059";
+  margin-right: 5px;
+  font-size: 13px;
+}
+
+.callout-ex-solution {
+  border-left-color: #ffe79a !important;
+}
+.callout-ex-solution > .callout-header {
+  background-color: #fff9e7 !important;
+}
+.callout-ex-solution > .callout-header::before {
+  font-family: "Font Awesome 5 Free";
+  content: "\f058";
+  margin-right: 5px;
+  font-size: 13px;
+}
+
```

Here is the code to add a functional chevron to each exercise callout. I've also included a line to avoid the callout header being highlighted every time it is clicked.

```suggestion
.callout-ex-prompt > .callout-header::after,
.callout-ex-hint > .callout-header::after,
.callout-ex-solution > .callout-header::after {
  font-family: "Font Awesome 5 Free";
  content: "\f054";
}

.callout-ex-prompt > .callout-header[aria-expanded="true"]::after,
.callout-ex-hint > .callout-header[aria-expanded="true"]::after,
.callout-ex-solution > .callout-header[aria-expanded="true"]::after {
  content: "\f078";
}

.callout-header {
  user-select: none;
}
```

---

Great, thank you! I modified this slightly to include an animated rotation instead of a switch of character